src/main.rs

```diff
@@ -211,3 +167,66 @@ fn main() -> Result<ExitCode, FlakeCheckerError> {
 
     Ok(ExitCode::SUCCESS)
 }
+
+#[cfg(feature = "allowed-refs")]
+#[derive(Parser)]
+struct Cli {
+    // Check to make sure that Flake Checker is aware of the current supported branches.
+    #[arg(long, hide = true)]
```

Comment:
Why are these hidden?


Also it seems like this should be an enum rather than flags if it's "either-or"

Reply:
These are meant only for "administrative" purposes, i.e. they're not meant to be used by anyone but our CI